strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ArtifactEntryServiceImpl.java

```diff
@@ -59,15 +69,27 @@
         ArtifactCoordinates coordinates = entity.getArtifactCoordinates();
         if (coordinates == null)
```

Comment:
yeah, I see that this is like in `master` but currently we shouldn't have any `AritfactEntry` without `ArtifactCoordinates`.
So can you please drop this `if` statement and check that we have `not null` constraint for `ArtifactEntry.artifactCoordinates`?